hearts/static/room.js

```diff
@@ -7,15 +7,35 @@ function apiCard(displayCard) {
     return displayCard[1] + displayCard[0];
 }
 
-// Will remove dummy data once we pull state from the API
+// API returns data of the form
+/*
+{
+    'players': [str],
+    'max_points': int,
+    'rounds': [
+        {
+            'players': str,
+            'direction': str,
+            'turn': int,
+            'hands': {str: [str]},
+            'tricks': [[str]],
+            'hearts': self.hearts_broken,
+            'current_scores': {str: int},
+            'final_scores': {str: int},
+            'is_over': boolean,
+        }, ...
+    ],
+    'round_number': int,
+    'scores': {str: int},
```

Comment:
According to hearts/game/hearts.py, `'scores'` is a list of scores from each completed round.

So the line should be 
`'scores': [{str: int}],